src/main/java/frc/team2412/robot/subsystems/LauncherSubsystem.java

```diff
@@ -76,63 +82,102 @@ public class LauncherSubsystem extends SubsystemBase {
+
 	// Constructor
 	public LauncherSubsystem() {
 
 		// MOTOR INSTANCE VARIBLES
 		// motors
 		launcherTopMotor = new CANSparkFlex(Hardware.LAUNCHER_TOP_MOTOR_ID, MotorType.kBrushless);
 		launcherBottomMotor = new CANSparkFlex(Hardware.LAUNCHER_BOTTOM_MOTOR_ID, MotorType.kBrushless);
-		launcherAngleMotor = new CANSparkFlex(Hardware.LAUNCHER_ANGLE_MOTOR_ID, MotorType.kBrushless);
+		launcherAngleOneMotor =
+				new CANSparkFlex(Hardware.LAUNCHER_PIVOT_ONE_MOTOR_ID, MotorType.kBrushless);
+		launcherAngleTwoMotor =
+				new CANSparkFlex(Hardware.LAUNCHER_PIVOT_TWO_MOTOR_ID, MotorType.kBrushless);
 		// encoders
 		launcherTopEncoder = launcherTopMotor.getEncoder();
 		launcherBottomEncoder = launcherBottomMotor.getEncoder();
-		launcherAngleEncoder = launcherAngleMotor.getAbsoluteEncoder(Type.kDutyCycle);
+		launcherAngleEncoder = launcherAngleOneMotor.getAbsoluteEncoder(Type.kDutyCycle);
 
 		// PID controllers
 		// Create launcherTopPIDController and launcherTopMotor]
 		launcherTopPIDController = launcherTopMotor.getPIDController();
 		launcherTopPIDController.setFeedbackDevice(launcherTopEncoder);
 		launcherBottomPIDController = launcherBottomMotor.getPIDController();
 		launcherBottomPIDController.setFeedbackDevice(launcherBottomEncoder);
-		launcherAnglePIDController = launcherAngleMotor.getPIDController();
-		launcherAnglePIDController.setFeedbackDevice(launcherAngleEncoder);
+		launcherAngleOnePIDController = launcherAngleOneMotor.getPIDController();
+		launcherAngleOnePIDController.setFeedbackDevice(launcherAngleEncoder);
+		// launcherAngleTwoPIDController = launcherAngleTwoMotor.getPIDController();
+		// launcherAngleTwoPIDController.setFeedbackDevice(launcherAngleEncoder);
+
 		Shuffleboard.getTab("Launcher")
-				.add(new SparkPIDWidget(launcherAnglePIDController, "launcherAnglePIDController"));
+				.add(new SparkPIDWidget(launcherAngleOnePIDController, "launcherAngleOnePIDController"));
+		// Shuffleboard.getTab("Launcher")
+		//		.add(new SparkPIDWidget(launcherAngleTwoPIDController, "launcherAngleTwoPIDController"));
 		Shuffleboard.getTab("Launcher")
 				.add(new SparkPIDWidget(launcherTopPIDController, "launcherTopPIDController"));
 		Shuffleboard.getTab("Launcher")
 				.add(new SparkPIDWidget(launcherBottomPIDController, "launcherBottomPIDController"));
+
+		configMotors();
 	}
 
 	public void configMotors() {
 		launcherTopMotor.restoreFactoryDefaults();
 		launcherBottomMotor.restoreFactoryDefaults();
-		launcherAngleMotor.restoreFactoryDefaults();
+		launcherAngleOneMotor.restoreFactoryDefaults();
+		launcherAngleTwoMotor.restoreFactoryDefaults();
 		// idle mode (wow)
 		launcherTopMotor.setIdleMode(IdleMode.kCoast);
 		launcherBottomMotor.setIdleMode(IdleMode.kCoast);
-		launcherAngleMotor.setIdleMode(IdleMode.kBrake);
+		launcherAngleOneMotor.setIdleMode(IdleMode.kBrake);
+		launcherAngleTwoMotor.setIdleMode(IdleMode.kBrake);
 		// inveritng the bottom motor lmao
 		launcherBottomMotor.setInverted(true);
```

Comment:
Now that we see the motors installed, it does appear we need to invert one of the angle motors so they do not work against each other and break pivot.  The one piece they told us not to break.

Reply:
Fixed.